fiasko_bro/code_validator.py

```diff
@@ -244,6 +264,12 @@ def validate(self, repo_path, original_repo_path=None, **kwargs):
         self.validator_arguments.update(kwargs)
         self.validator_arguments['whitelists'] = self.whitelists
         self.validator_arguments['blacklists'] = self.blacklists
+        if is_repos_too_large(
+            repo_path,
+            original_repo_path,
+            self.validator_arguments['max_num_of_py_files']
+        ):
+            return [('repo is too large', '')]
```

Comment:
This part is actually done well, but I don't like how it spans 5 lines. Assign `self.validator_arguments['max_num_of_py_files']` to a variable a line before the `if` and make it so that condition would be on one line. 

Again, don't worry too much about 80-char limit. It's there to improve readability, and should be ignored if it doesn't serve its goal.